chrome/content/api/LegacyPrefs/implementation.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable object-shorthand */
```

Comment:
Please use the latest version from:
https://github.com/thundernest/addon-developer-support/tree/master/auxiliary-apis/LegacyPrefs